integration_tests/fixtures/assessmentData.json

```diff
@@ -39,7 +39,7 @@
       "outlineOfDiscussion": "Some discussion"
     }
   },
-  "make-a-decision": { "make-a-decision": { "decision": "accept" } },
+  "make-a-decision": { "make-a-decision": { "decision": "accept", "rejectReason": "some rejected comment" } },
```

Comment:
Given the decision here is "accept", I'd guess the reason should be `undefined` or an empty string?

Reply:
this is the testdata being used for all test cases accept and reject

Reply:
does it help if we name the "rationale" property in a decision-agnostic way e.g.

```
"make-a-decision": { "make-a-decision": { "decision": "accept", "decisionRationale": "some rationale for the decision" } 
```

Reply:
That probably makes more sense given you can still provide a rationale when accepting a placement